queries/dune_v2/period_slippage.sql

```diff
@@ -136,13 +133,12 @@ incoming_and_outgoing as (
         left outer join tokens.erc20 t
             on i.token = t.contract_address
             and blockchain = 'ethereum'
-    where
+    where tx_hash not in (select tx_hash from exluded_batches)
       -- We exclude settlements that have zero AMM interactions and settle several trades,
       -- as our query is not good enough to handle these cases accurately.
       -- Settlements with dex_swaps = 0 and num_trades = 0 can be handled in the following
       -- and we want to consider them in order to filter out illegal behaviour
-        (dex_swaps = 0 and num_trades < 2) or dex_swaps > 0
-        and tx_hash not in (select tx_hash from exluded_batches)
+      and ((dex_swaps = 0 and num_trades < 2) or dex_swaps > 0)
```

Comment:
This was a logical error in our constraints. Our query before was
```
where (A and B) or C    <-- These should have been enclosed in brackets
and D
```
and is changed to 
```
where D
and ((A and B) or C)
```
cf this logical table (proof): https://dune.com/queries/1656725


This is a bit messy because of a rearrangement, but it is better this way.